### DISCUSSION TITLE
photobooth

### DISCUSSION THREAD

**Discussion by Aikigamer** [TarrySingh/Artificial-Intelligence-Deep-Learning-Machine-Learning-Tutorials#404] [Q&A]
Hi there, i am working on a project using Raspberry Pi (ideally) to integrate Photo manipulative AI into a photobooth to help manipulate the photo in creative ways. do you think i will be able to do something like this?

**Comment by tashamakseh426-create** [comment]
That sounds like a genuinely fun Raspberry Pi project. I’d say it’s definitely feasible, especially if the “creative manipulation” is kept to a few well-defined effects rather than trying to run a huge generative model locally. A lightweight pipeline could handle capture, preprocessing, and basic transformations, while heavier AI inference runs remotely when needed. Good datasets and image collection also matter a lot here; I’ve found [this](https://unidata.pro/data-collection/) site useful as a reference for that side of AI projects.
